### PR TITLE
[FIX] Improve the design of readme file and add contributors section.

### PR DESCRIPTION
**Pull request by Freakyab**
[FIX] Improve the design of readme file and add contributors section. 

Resolve issue 
#53

### CODE REVIEW

**Comment by gauravsingh1281** [comment]
@Freakyab Show me the screenshot

**Comment by Freakyab** [comment]
> @Freakyab Show me the screenshot

https://github.com/Freakyab/Rentalog-Rent-Management-Web-App-For-Landlords/blob/main/README.md

**Comment by gauravsingh1281** [comment]
> > @Freakyab Show me the screenshot
> 
> https://github.com/Freakyab/Rentalog-Rent-Management-Web-App-For-Landlords/blob/main/README.md

Don't change the text or information which are written in the original readme file. Try to redesign it and make it more catchy or engaging.

**Comment by Freakyab** [comment]
> > > @Freakyab Show me the screenshot
> > 
> > 
> > https://github.com/Freakyab/Rentalog-Rent-Management-Web-App-For-Landlords/blob/main/README.md
> 
> Don't change the text or information which are written in the original readme file. Try to redesign it and make it more catchy or engaging.

for making it catchy, I have added keywords otherwise the content is same. Also added Contributors names to readme file.

**Review by gauravsingh1281 — CHANGES_REQUESTED** [review]
This text are different from original one

File: README.md [hunk 1]
```diff
@@ -1,76 +1,85 @@
...
-    <li>Username - demo@rentalog.in</li>
-    <li>Password - @1a2b3c4d5E</li>
-  </ul>
-  </li>
-</ul>
-<h2>License</h2>
-<p>Rentalog is released under the <strong>MIT License</strong>. By contributing to this project, you agree to provide your contributions under the same license.</p>
+# Rentalog - Rent Management Web App For Landlords🌐
+
+<p>Welcome to Rentalog, the avant-garde web application designed to elevate the way landlords manage their rental properties. Rentalog empowers landlords by seamlessly organizing crucial information about renters, ensuring a secure and organized repository for quick and easy access. Beyond tenant data, Rentalog simplifies rent collection and tracking, providing landlords with real-time insights into their property earnings. It's the ultimate solution for landlords who value efficiency, organization, and informed decision-making.</p>
```
Comment (gauravsingh1281) [inline README.md:3]: this

File: README.md [hunk 2]
```diff
@@ -1,76 +1,85 @@
...
+
+---
+
+## Features 🚀
```
Comment (gauravsingh1281) [inline README.md:7]: this

**Review by gauravsingh1281 — CHANGES_REQUESTED** [review]
don't touch the text which is written in the original file

**Comment by gauravsingh1281** [comment]
closes via #106

**Pull request closed without merging**